src/components/FileUploader/index.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import Icon from '../Icon';
+import Spinner from '../Spinner';
+import UploaderItem from '../UploaderItem';
+import { StyledFileUploader, FileUploaderLabel } from './styles';
```

Comment:
Why is the first component imported prefixed by `Styled` when the second is not?

Reply:
I can't name it FileUploader since that's the name of the component itself. We used to prefix `Styled` to everything in the `admin` before and that got annoying real fast, so now we do it only when necessary.

I'd be happy to change it to any other name if it makes more sense

Reply:
No ok, I was just wondering 🙂